src/dispatch/messaging.py

```diff
@@ -496,6 +500,14 @@ class MessageType(str, Enum):
     {"title": "Incident Commander Re-Added", "text": INCIDENT_COMMANDER_READDED_DESCRIPTION}
 ]
 
+INCIDENT_CLOSED_INFORMATION_REVIEW_REMINDER_NOTIFICATION = [
+    {
+        "title": "{{name}} Incident - Information Review Reminder",
+        "title_link": "{{ticket_weblink}}",
```

Comment:
This link would go to Jira afaik. Is that what we want? Or do we always want to take them to the Dispatch UI (since we're asking them to modify it there).

Reply:
I'll change it to the Dispatch UI and work on adding the ability to link to tickets directly in Dispatch.